HSDS/Common/ResourceBase.h

```diff
@@ -14,7 +14,7 @@ class ErrorResponse {
 public:
   ErrorResponse()
     : status_(httpserver::http::http_utils::http_bad_request)
-    , message_("OK")
+    , message_("Bad Request")
```

Comment:
Applying to main branch https://github.com/UnityFoundation-io/CommunityUtils/pull/19